File: CS/Custom Wizard/Data Wrapper Classes/ConnectionWrapper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.DataAccess.ConnectionParameters;
using DevExpress.Mvvm.POCO;

namespace CustomDataSourceWizard {

    public class ConnectionWrapper {
        protected ConnectionWrapper() {
        }
        public static ConnectionWrapper Create() {
            return ViewModelSource.Create(() => new ConnectionWrapper());
        }
        public virtual string ConnectionName {
            get;
            set;
        }
        public virtual DataConnectionParametersBase ConnectionParameters {
            get;
            set;
        }
    }
}

File: VB/Custom Wizard/Data Wrapper Classes/ConnectionWrapper.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.DataAccess.ConnectionParameters
Imports DevExpress.Mvvm.POCO

Namespace CustomDataSourceWizard

	Public Class ConnectionWrapper
		Protected Sub New()
		End Sub
		Public Shared Function Create() As ConnectionWrapper
			Return ViewModelSource.Create(Function() New ConnectionWrapper())
		End Function
		Public Overridable Property ConnectionName() As String
		Public Overridable Property ConnectionParameters() As DataConnectionParametersBase
	End Class
End Namespace
